File: bottle-deposits/solution/Dan90git/bottle-deposits.py
recycled_big = input('How many bottles that hold one liter or less do you have?')
recycled_little = input('how many bottles that hold more than a liter do you have?')

big_ones = float(recycled_big)
little_ones = float(recycled_little)

def refund(recycled_big, recycled_little):
    big = big_ones * 0.25
    little = little_ones * 0.10
    return round(big * little, 2)

print("if you will refund this containers you'll get " + str(refund(recycled_big, recycled_big)) +" $")

File: projects/m1/002-bottle-deposits/solutions/vittorioMenicanti/bottleDeposits.js


//bottle deposits:

const SmallContainer = 0.10;
const BigContainer = 0.25;
let totalDepositSmall;
let totalDepositBig;


const totalSmallContainer = +prompt("Insert how many containers holding one liter or less do you return?");

const totalBigContainer = +prompt("Insert how many containers holding more than one liter do you return?");

totalDepositSmall = (SmallContainer) * (totalSmallContainer);

totalDepositBig = (BigContainer) * (totalBigContainer);


alert( `Your refund is ${(totalDepositSmall + totalDepositBig).toFixed(2)}$` );